src/System.Runtime.Serialization.Formatters/tests/BinaryFormatterTestData.cs

```diff
@@ -11,15 +11,29 @@
 using System.Numerics;
 using System.Reflection;
 using System.Text;
-using Xunit;
 
 namespace System.Runtime.Serialization.Formatters.Tests
 {
     public partial class BinaryFormatterTests
     {
-        // *** AUTO UPDATED BLOBS ***
         private static DateTime s_fixedTimestamp = DateTime.Parse("11/3/1989 04:50:29 AM", CultureInfo.InvariantCulture.DateTimeFormat);
 
+        // Do not move this method without updating the Exception blobs.
+        // They contain the class name and line number of this method in their StackTrace.
+        private static T ThrowWithStackTrace<T>(T exception)
```

Comment:
it would be nice to set some fields like HelpUrl, Data, HResult which are presumably just null.